docs/reference/packages.rst

```diff
@@ -56,6 +56,24 @@ If no json file is specified, then "package.json" is implicitly added::
     >>> mip.install("github:org/repo")  # Uses default branch of that repo
     >>> mip.install("github:org/repo", version="branch-or-tag")
 
+The URL can also start with ``gitlab:`` as a simple way of pointing to content
```

Comment:
Thanks for adding docs.

This section mostly just duplicates the above though, and because the gitlab support works in exactly the same way at github, maybe just say

```
The URL can also start with ``gitlab:`` and works in exactly the same way as GitHub URLs described above ``gitlab:org/repo[/path]``, including support for branches.
```